presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduNodePartitioningProvider.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kudu;
+
+import io.prestosql.spi.connector.BucketFunction;
+import io.prestosql.spi.connector.ConnectorBucketNodeMap;
+import io.prestosql.spi.connector.ConnectorNodePartitioningProvider;
+import io.prestosql.spi.connector.ConnectorPartitionHandle;
+import io.prestosql.spi.connector.ConnectorPartitioningHandle;
+import io.prestosql.spi.connector.ConnectorSession;
+import io.prestosql.spi.connector.ConnectorSplit;
+import io.prestosql.spi.connector.ConnectorTransactionHandle;
+import io.prestosql.spi.connector.SchemaTableName;
+import io.prestosql.spi.type.Type;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.function.ToIntFunction;
+import java.util.stream.IntStream;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.prestosql.spi.connector.ConnectorBucketNodeMap.createBucketNodeMap;
+import static java.util.Objects.requireNonNull;
+
+public class KuduNodePartitioningProvider
+        implements ConnectorNodePartitioningProvider
+{
+    private final KuduClientSession clientSession;
+
+    @Inject
+    public KuduNodePartitioningProvider(KuduClientSession clientSession)
+    {
+        this.clientSession = requireNonNull(clientSession, "clientSession is null");
+    }
+
+    @Override
+    public List<ConnectorPartitionHandle> listPartitionHandles(
+            ConnectorTransactionHandle transactionHandle,
+            ConnectorSession session,
+            ConnectorPartitioningHandle partitioningHandle)
+    {
+        KuduPartitioningHandle handle = (KuduPartitioningHandle) partitioningHandle;
+        int bucketCount = handle.getBucketCount();
+        return IntStream.range(0, bucketCount).mapToObj(KuduPartitionHandle::new).collect(toImmutableList());
+    }
+
+    @Override
+    public ConnectorBucketNodeMap getBucketNodeMap(ConnectorTransactionHandle transactionHandle,
```

Comment:
Can we move each argument to a new line ?

Reply:
Ok, will do!